python/dgl/model_zoo/chem/sch.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding:utf-8 -*-
+
```

Comment:
Add a line `pylint: disable=C0103, C0111`, otherwise we cannot pass the CI check.